moda-contracts/src/Catalog.sol

```diff
@@ -83,7 +85,7 @@ contract Catalog is ICatalog, AccessControlUpgradeable {
         $._version = version;
         $._modaRegistry = modaRegistry;
         $._membership = membership;
-
+        $._management = IOfficialModaContracts(modaRegistry).getManagement();
```

Comment:
We don't need to store the management value. We can look it up dynamically when needed. It may change if we roll out a new version so we want ModaRegistry to be the source of truth. 

`getManagement` should return `IManagement`